### PR TITLE
Add Learn Architecture section and related pages.

### PR DESCRIPTION
**Pull request by meganskye**

### CODE REVIEW

**Review by gluneau — APPROVED** [review]
It's good.

Although, I'm not a fan of warranties. ;-)

File: docs/build/Introduction/astar_family.md [hunk 1]
```diff
@@ -6,26 +6,41 @@ sidebar_position: 5
...
+- zkEVM node components: aggregator, approve service, sequencer and sequence sender, synchronizer
+- L1 network (mock)
+- Prover
+- Explorers L1, L2
+- JSON RPC explorer
+- L2 gas pricer
+- DAC: data availability service, dac setup committee
+- zkEVM bridge service and UI
+
+For more information about zkEVM nodes including the rollup-based zKatana version, check out the [zkEVM Build section](/docs/build/zkEVM/zk-node/).
```
Comment (shaunxw) [inline docs/build/Introduction/astar_family.md:39]: Please remove zkEVM part. There is no local node concept for L2.

File: docs/build/Introduction/astar_family.md [hunk 2]
```diff
@@ -34,28 +49,34 @@ The Shibuya native token symbol is SBY.
...
-Rococo is a test Relay Chain used by the Polkadot & Kusama communities. Astar team has deployed a parachain to it called Rocstar, which is mainly used for cross-chain integrations with other teams in the ecosystem. To obtain test tokens for Rocstar, please contact Astar team on Discord.
+### zKyoto (Sepolia)
 
-The Rocstar native token symbol is ROC.
+zKyoto is a Validium-based network connected to the Ethereum Sepolia testnet and Polygon AggLayer. It is the testing and proving ground for dApps deployed on the Astar zkEVM.
 
 ## Mainnets
 
-Astar has two mainnets, like most parachains in the Polkadot ecosystem. One on Kusama Relay Chain, and the other on Polkadot Relay chain.
+Astar has three mainnets. One parachain on Kusama, One parachain on Polkadot, and a zkEVM-based Validium network on Ethereum Layer 2.
```
Comment (shaunxw) [inline docs/build/Introduction/astar_family.md:62]: I suggest we change the ordering of the mainnets intro, to 1. Astar zkEVM 2. Astar Parachain 3. Shiden Parachain

File: docs/build/Introduction/astar_family.md [hunk 3]
```diff
@@ -34,28 +49,34 @@ The Shibuya native token symbol is SBY.
...
+### Astar Parachain (Substrate)
 
-By now you may have already guessed that Astar network is a parachain on Polkadot Relay chain.
+By now you may have guessed that Astar parachain is connected to Polkadot Relay chain.
 
 The Astar native token symbol is ASTR.
 
+### Astar zkEVM
+
+Astar zkEVM is a Layer 2 scaling solution for Ethereum, powered by Polygon AggLayer. Technically referred to as a Validium, the zkEVM combines zk rollups with modular data availability, synthesizing the best features of each.
```
Comment (shaunxw) [inline docs/build/Introduction/astar_family.md:78]:
```suggestion
Astar zkEVM is an Ethereum Layer 2 network, powered by Polygon CDK. Technically referred to as Validium, it combines zk rollup technology with modular data availability, synthesizing the best features of each.
```

File: docs/build/Introduction/img/fig3.jpg
Comment (shaunxw) [inline docs/build/Introduction/img/fig3.jpg:1]: Why there is a DeSpread logo in the image?

File: docs/learn/architecture/astar-zkevm.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+title: Astar zkEVM
+sidebar_position: 2
+---
+
+# Overview
+The Astar zkEVM is a Layer 2 scaling solution for Ethereum connected to Polygon Aggregation Layer (AggLayer) for shared interoperability and data availability, otherwise known as a Validium. Smart contracts that run on the Ethereum Virtual Machine also run on Astar zkEVM. The zkEVM boasts the following key features:
+
+- **Higher TPS than Ethereum or Astar Substrate EVM** - Leveraging zk rollup architecture, transactions are parallelized on Layer 2 and submitted on-chain to Layer 1 in batches, supercharging web3 games and DeFi applications requiring high performance.
```
Comment (shaunxw) [inline docs/learn/architecture/astar-zkevm.md:9]: This is wrong. Astar zkEVM doesn't have a higher TPS.

File: docs/learn/architecture/astar-zkevm.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+- **Lower Transaction Fees compared to Ethereum** - Due to the transaction batching, as explained above.
+- **Full EVM-equivalence** - Not only EVM compatibility; Equivalence. Smart contracts that work on Ethereum also work on Astar zkEVM. See the [smart contract section](/docs/build/zkEVM/smart-contracts/) for more information.
+- **Native Account Abstraction** - The Astar zkEVM provides native features designed to revolutionize the end-user experience, and make it seamless. See the [Account Abstraction section](/docs/build/zkEVM/integrations/account-abstraction/) to learn more about how to refine the end-user experience.
+- **Recognized Partners** - Established names and brands that developers trust power the Astar zkEVM. See the [integrations section](/docs/build/zkEVM/integrations/) for more information about 3rd party service providers.
```
Comment (shaunxw) [inline docs/learn/architecture/astar-zkevm.md:13]: The term *partners* shouldn't be used. Partners means 50% and 50% liability.

File: docs/learn/architecture/astar-zkevm.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+---
+title: Astar zkEVM
+sidebar_position: 2
+---
+
+# Overview
+The Astar zkEVM is a Layer 2 scaling solution for Ethereum connected to Polygon Aggregation Layer (AggLayer) for shared interoperability and data availability, otherwise known as a Validium. Smart contracts that run on the Ethereum Virtual Machine also run on Astar zkEVM. The zkEVM boasts the following key features:
```
Comment (shaunxw) [inline docs/learn/architecture/astar-zkevm.md:7]:
```suggestion
The Astar zkEVM is a Layer 2 scaling solution for Ethereum connected to Polygon Aggregation Layer (AggLayer) for shared interoperability. It's a modular Validium blockchain. Smart contracts that run on the Ethereum Virtual Machine also run on Astar zkEVM. The zkEVM boasts the following key features:
```

File: docs/learn/architecture/img/fig1.png
Comment (shaunxw) [inline docs/learn/architecture/img/fig1.png:1]: Astar zkEVM is not zk-Rollup

File: docs/learn/architecture/img/fig3.jpg
Comment (shaunxw) [inline docs/learn/architecture/img/fig3.jpg:1]: Why DeSpread logo?

File: docs/learn/img/fig3.jpg
Comment (shaunxw) [inline docs/learn/img/fig3.jpg:1]: Same here.

File: docs/learn/networks.md [hunk 7]
```diff
@@ -23,9 +23,25 @@ For more information about Swanky Node, check out the [Swanky Suite](https://doc
...
+
+- zkEVM databases: data node, event, explorer L1 and L2, pool, state, and bridge service
+- zkEVM node components: aggregator, approve service, sequencer and sequence sender, synchronizer
+- L1 network (mock)
+- Prover
+- Explorers L1, L2
+- JSON RPC explorer
+- L2 gas pricer
+- DAC: data availability service, dac setup committee
+- zkEVM bridge service and UI
```
Comment (shaunxw) [inline docs/learn/networks.md:38]: A lot of wrong info. Please refer to https://github.com/0xPolygon/cdk-validium-node?tab=readme-ov-file#key-components and re-write

File: docs/learn/architecture/img/fig3.jpg
Comment (meganskye) [reply docs/learn/architecture/img/fig3.jpg:1]: this is a placeholder

File: docs/learn/img/fig3.jpg
Comment (meganskye) [reply docs/learn/img/fig3.jpg:1]: again, this is a placeholder. nate is working on the updated image.

File: docs/learn/networks.md (hunk 7 above)
Comment (meganskye) [reply docs/learn/networks.md:38]: this is taken directly from https://docs.astar.network/docs/build/zkEVM/zk-node/setup-local-validium-node so one of our devs will need to provide the correct information for both sections.

Comment (meganskye) [reply docs/learn/networks.md:38]: for now, this section has been removed.

File: docs/build/Introduction/img/fig3.jpg
Comment (meganskye) [reply docs/build/Introduction/img/fig3.jpg:1]: updated

**Pull request merged**